templates/web/pages/bmviii_home_temp.tmpl

```diff
@@ -0,0 +1,265 @@
+            <div class="bmviii-left-accordion-content">
+                <p>
+                    No problem! Come to BoilerMake to learn a new skill and meet people. We'll have tech talks, a Slack workplace to ask questions,
+                    and engineers from sponsoring companies to help you build something you can be proud of. This is your best chance
+                    to learn alongside so many other driven hackers.
+                </p>
+            </div>
+            <button class="bmviii-left-accordion">How large can my team be?</button>
+            <div class="bmviii-left-accordion-content">
+                <p>
+                    You can have a maximum of 4 people on your team.
+                </p>
+            </div>
+        </div>
+        <div class="column is-half"; style = "padding-top 120px;">
+            <button class="bmviii-right-accordion">What are the requirements to attend?</button>
+            <div class="bmviii-right-accordion-content">
+                <p>
+                    BoilerMake is open to all undergraduate college students who are of age 18 or older.
+                </p>
+            </div>
+            <button class="bmviii-right-accordion">My team was accepted and I wasn't - can I attend?</button>
+            <div class="bmviii-right-accordion-content">
+                <p>
+                    Feel free to email team@boilermake.org for info about our waitlist, and we will do our best to accommodate you as space opens up at our event.
+                </p>
+            </div>
+            <button class="bmviii-right-accordion">Do you allow walk-ins and last minute registration?</button>
+            <div class="bmviii-right-accordion-content">
+                <p>
+                    We accept walk-ins based on availability after hacker check-in.
+                    We will announce more details closer to the event!
+                </p>
+            </div>
+            <button class="bmviii-right-accordion">I missed the deadline to apply - can I attend?</button>
+            <div class="bmviii-right-accordion-content">
+                <p>
+                    We set a deadline to ensure that all applications are given equal consideration to attend BoilerMake.
+                    Feel free to email team@boilermake.org for info about our waitlist, and we will do our best to accommodate you as space opens up at our event.
+                </p>
+            </div>
+            <button class="bmviii-right-accordion">What if I have any other questions?</button>
+            <div class="bmviii-right-accordion-content">
+                <p>
+                    Please email us at <a href='mailto:team@boilermake.org'>team@boilermake.org</a>!
+                </p>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="section has-background-image" id="testimonials" style='background-image: url({{ static_path "bmviii_testimonial_full_background.svg" }})'>
```

Comment:
The umbrella gets cut off by something:
![Screen Shot 2020-10-05 at 21 22 44](https://user-images.githubusercontent.com/10781907/95148268-87a96f00-0748-11eb-9304-93de208852a7.png)

Reply:
This is the issue with the full height of the background image not showing. To get it to show, I would have to increase the margin-top of the container but that brings on a lot more problems